src/renderer_mtl.mm

```diff
@@ -15,6 +15,12 @@
 #	include <Cocoa/Cocoa.h>
 #endif
 
+#if BX_PLATFORM_VISIONOS
+#   define BGFX_GET_SWAPCHAIN_PIXEL_FORMAT(swapChain)   (MTLPixelFormatBGRA8Unorm_sRGB)
+#else
```

Comment:
Make function `getSwapChainPixelFormat` instead of macros.

Reply:
Refactored it to a funciton